gravitee-apim-e2e/api-test/apis/api-definition.spec.ts

```diff
@@ -749,9 +749,12 @@ afterAll(async () => {
   });
   expect(apiStoppedResponse.raw.status).toEqual(204);
   const apiPagesApi = new APIPagesApi(configuration);
+  const deleteApiPages = [];
   for (const pageId of pageIds.reverse()) {
-    await apiPagesApi.deleteApiPage({ orgId, envId, api, page: pageId });
+    deleteApiPages.push(apiPagesApi.deleteApiPage({ orgId, envId, api, page: pageId }));
   }
 
-  await apisApi.deleteApi({ orgId, envId, api: api });
+  await Promise.all(deleteApiPages);
```

Comment:
Is it ok to delete everything in parallel? If so then we can simplify a bit L752:

```ts
  const deleteApiPages = pageIds.map(page => apiPagesApi.deleteApiPage({ orgId, envId, api, page }))

  await Promise.all(deleteApiPages);
```